import-service/pubsub.tf

```diff
@@ -0,0 +1,51 @@
+resource "google_pubsub_topic" "import-service-pubsub-topic" {
+  project = module.import-service-project.project_name
+  name = "import-service-notify"
+}
```

Comment:
shouldn't this be `import-service-notify-${var.env}`? (or some other syntax flavor of $ENV)

Reply:
no. each import service instance lives in its own google project. the google project has the name of the env in it; there's no need to put the env in the topic name too, because there will only be one topic per project.

Reply:
But this isn't what Rawls uses: https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/rawls/rawls.conf.ctmpl#L341. You previously said "in the long run import service in dev will be deployed using the terraform i'm writing and i'll make sure that rawls configs are adjusted to point to the right queues as part of that" ... will there be a change to Rawls config, or a change here?

Reply:
uuuurgh. good catch. i'll make a change to the rawls conf